### DISCUSSION TITLE
Welcome to bluelinky Discussions!

### DISCUSSION THREAD

**Discussion by Hacksore** [Hacksore/bluelinky#178] [General]
## 👋 Welcome to bluelinky Discussions!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Boxana** [comment]
Hallo,
wird es das ach wieder für Node Red geben?
Auf eine antwort würde ich mich freuen.
Gruß
Christian

**Comment by PierreLevres** [comment]
Ja!

**Comment by Boxana** [comment]
ich bin kein Programmierer würde  aber dennoch gerne dazu beitragen.
Gibt es ein donation link für dieses Projekt?

**Reply by PierreLevres** [reply]
My suggestion is to discuss in English so everyone can participate. Then @Hacksore could answer the question about donations.

**Comment by Boxana** [comment]
I have now set up a donation. I'm just not sure if it goes to the right people :-).
 If not then that's stupid.

That was the subject of the confirmation email.
You're now sponsoring @Hacksore!

is that OK?

**Reply by Hacksore** [reply]
> I have now set up a donation. I'm just not sure if it goes to the right people :-). If not then that's stupid.
> 
> That was the subject of the confirmation email. You're now sponsoring @Hacksore!
> 
> is that OK?

Thanks very much - Really the support goes to all the people who dedicate their time to contributing to this project, developer or not.

**Comment by Boxana** [comment]
Hi there,
keep getting this message.
Only a restart brings temporary improvement.

Error HTTP status 400 sending put-request with body 'ManagedBluelinkyError: @EuropeVehicle.status: [401] Unauthorized on [GET] https://prd.eu-ccapi.hyundai.com:8080/api/v2/spa/vehicles/xxxxxxxxxxxx/status/latest - {"retCode":"F","resCode":"4017","resMsg":"Invalid request value - Invalid Application Id. - 1013.1579999923706","msgId":"056cedbb-da25-42e4-xxxxxx-2b30167f443e"}'

**Comment by iProDeveloping** [comment]
does this work for kia and genesis too?

**Reply by Hacksore** [reply]
Kia should be but no one has reversed the genesis API.

**Comment by ytsopelas** [comment]
Hi - any chance for a Genesis HA integration? Happy to help with testing. I'm in the UK. Thanks
